### PR TITLE
Change guide hash changes to replaceState;

### PR DESCRIPTION
**Pull request by steven1046**
so it doesn't consume the browser history stack with every step selected

#### Were the changes tested on
- [x] Firefox (Desktop)
- [x] Safari (Desktop)
- [x] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] Dymanic Accessability Plugin (DAP)

### CODE REVIEW

**Comment by steven1046** [comment]
It's having trouble going back in Safari only, let me figure out why before you review it @mbroz2.

**Comment by steven1046** [comment]
Nevermind @mbroz2, I was viewing a different port on my Safari browser, it is also working on Safari.

File: src/main/content/_assets/js/toc.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ function TOCEntryClick(liElement, event) {
         // So, this history.pushState will allow us to set the URL without
```
Comment (mbroz2) [inline src/main/content/_assets/js/toc.js]: would be good to update this comment to .replaceState

**Comment by steven1046** [comment]
I updated the comment @mbroz2

**Pull request merged**